modules/troubleshooting-network-observability-loki-resource-exhausted.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
+= Troubleshooting Loki ResourceExhausted error
+Loki may return a "ResourceExhausted" error when network flow data sent by Network Observability exceeds the configured maximum message size. When using the Loki operator, this maximum message size is configured to 100 MiB. 
```

Comment:
```suggestion
:_content-type: PROCEDURE
[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
= Troubleshooting Loki ResourceExhausted error

Loki may return a `ResourceExhausted` error message when network flow data sent by Network Observability exceeds the configured maximum message size. If you are using the Loki Operator, this maximum message size is configured to 100 MiB. 
```

- Suggest putting `ResourceExhausted` in backticks if it's a literal name of an error
- "Error message" per the note at the end of https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages
- Suggest consider adding an attribute for the product name Network Observability
- Operator always needs to be capitalized
- Suggested editing for active over passive tense

Reply:
TY, excellent suggestions/comments. I really do need to make an attribute.